app/script/localization/translations/webapp-de.js

```diff
@@ -97,8 +97,9 @@ z.string.de.callAccept = 'Annehmen';
 z.string.de.callJoin = 'Beitreten';
 z.string.de.callChooseSharedScreen = 'Wähle einen Bildschirm aus';
 z.string.de.callParticipants = '{{number}} im Anruf';
+z.string.de.callNoCameraAccess = 'Kein Kamerazugriff';
 
-z.string.de.videoCallOverlayFitVideoLabel = 'Füllmodus der Videos kann per Doppelklick geändert werden';
+z.string.de.videoCallOverlayFitVideoLabel = 'Das Video per Doppelklick anpassen';
```

Comment:
@Astrid-Wire @infotexture This seems to be a weird German copy. This is supposed to be a hint and the chosen copy is not really clear to me. Now it looks like a somewhat random comment that you see when you move your mouse that is not connected to any UI need or user action,